otelib/backends/python/client.py

```diff
@@ -0,0 +1,93 @@
+"""Client for python backend."""
+from otelib.backends.python import (
+    DataResource,
+    Filter,
+    Function,
+    Mapping,
+    Transformation,
+)
+
+
+# pylint: disable=duplicate-code
+class OTEPythonClient:
```

Comment:
Agreed, but maybe let's make this a separate issue? Originally I wanted to have them both inherit form OTEClient, but that creates a circular reference (since it returns one its children, OTE{Python,Services}Client

Reply:
See issue https://github.com/EMMC-ASBL/otelib/issues/68